src/main/java/mate/academy/sevice/BookServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.sevice;
+
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.model.Book;
+
+public class BookServiceImpl implements BookService {
+    private final BookDao bookDao;
+
+    public BookServiceImpl(BookDao bookDao) {
+        this.bookDao = bookDao;
+    }
+
+    @Override
+    public Book create(Book book) {
+        return bookDao.create(book);
+    }
+
+    @Override
+    public Book get(Long id) {
+        return bookDao.findById(id)
+                .orElseThrow(() -> new RuntimeException("Can't find book with id " + id));
```

Comment:
Let's use a narrower exceprion type here. 
IllegalArgumentException for example.